src/renderer/components/distraction-settings/distraction-settings.js

```diff
@@ -68,7 +123,8 @@ export default Vue.extend({
       'updateHideLiveChat',
       'updateHideActiveSubscriptions',
       'updatePlayNextVideo',
-      'updateDefaultTheatreMode'
+      'updateDefaultTheatreMode',
+      'updateChannelBlockerList'
```

Comment:
I'm not seeing this function, along with a few others, in the project anywhere. Is this meant to exist?

Reply:
i think it's just a leftover.